simulariumio/data_objects/agent_data.py

```diff
@@ -263,3 +268,108 @@ def from_buffer_data(cls, buffer_data: Dict[str, Any]):
             draw_fiber_points=False,
             type_ids=type_ids,
         )
+
+    def append_agents(self, new_agents: AgentData):
+        """
+        Concatenate the new AgentData with the current data,
+        generate new unique IDs and type IDs as needed
```

Comment:
one question:  what if I want to append the same N agents to every time step (e.g. merging two whole trajectories that have same number of time steps) -- how can i ensure the agent ids from new_agents will stay the same across time if this is reassigning ids?   
Maybe this is a consideration for later but I bet it could come up easily.

Reply:
oh, i see that new_agents is a whole trajectory, not just one time step

Reply:
so new_agents is required to have same number of time steps as self?

Reply:
yeah it is. I want to add per frame functionality in a later PR, there's a lot of stuff that needs to be implemented and it makes sense to me to do it all at once?

Reply:
if you wanted to just add to some frames, for now you could have no agents in those frames in the added trajectory